### DISCUSSION TITLE
Using Keepass from Github on multiple sites?

### DISCUSSION THREAD

**Discussion by NilsRo** [keepassxreboot/keepassxc#11152] [Q&A]
Hi,

I have problems to setup Github as an authentication provider also for other sites like the OpenWRT forum. How this has to be done? Saving the Keypass only once or saving the credentials redundant for every site? I found a property called KP2A_URL that holding the site for authentication and KPEX_PASSKEY_RELYING_PARTY that containing Github as the provider.

Actually Keepass Plugin will fallback to browser authentication as it cannot locate the necessary Passkey. Adding forum.openwrt.org in the Browser-Integration does not solve this. Github.com is working fine...

Cheers
Nils

![msedge_2024-08-10 110748](https://github.com/user-attachments/assets/ea909a3b-6dcd-4aed-99aa-066a8c0f4004)

**Comment by droidmonkey** [comment]
When you register the passkey with the site it will set everything up for you, don't touch any of the attributes after that. Just re-do registration and it should work.

**Reply by NilsRo** [reply]
I have 2 options. Save the passkey twice or update the passkey still stored for GitHub. Is that the standard way to store the same passkey for every website again?

**Reply by droidmonkey** [reply]
Honestly have no idea what you are doing. You are going back and forth between GitHub and OpenWRT.

![image](https://github.com/user-attachments/assets/18c2acc4-1018-4209-a666-4187eb437666)

**Reply by NilsRo** [reply]
My mistake...I thought the passkey button is also provided by GitHub and tried to use the Github key there which failed. Thanks for the tipp, under security I found the section to set a passkey for the forum itself. Now it is working like expected.
